scripts/extract-paired-reads.py

```diff
@@ -94,11 +94,11 @@ def get_parser():
     parser.add_argument('-d', '--output-dir', default='', help='Output '
                         'split reads to specified directory. Creates '
                         'directory if necessary')
-    parser.add_argument('--output-paired', '-p', metavar="filename",
+    parser.add_argument('-p', '--output-paired', metavar="filename",
```

Comment:
There are many scripts where the long option is defined before the short option. Any reason for singling out this script?